assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -56,26 +58,44 @@ const Media3pCategories = ({
   deselectCategory,
 }) => {
   const [isExpanded, setIsExpanded] = useState(false);
+
+  function renderCategories() {
+    return categories.map((e) => {
+      return (
+        <CategoryPill
+          isSelected={false}
+          key={e.id}
+          title={e.displayName}
+          onClick={() => {
+            setIsExpanded(false);
+            selectCategory(e.id);
+          }}
+        />
+      );
+    });
+  }
+
+  function renderSelectedCategory() {
+    const selected = categories.find((e) => e.id === selectedCategoryId);
+    return (
+      <CategoryPill
+        isSelected={true}
+        key={selected.id}
+        title={selected.displayName}
+        onClick={() => deselectCategory()}
+      />
+    );
+  }
```

Comment:
Perhaps these 2 should be the same function, receiving `selected` as a param

Reply:
I kind of thought this might make it more readable. Changed anyway.